content/fast/using-bundlesize-with-travis-ci/index.md

```diff
@@ -0,0 +1,232 @@
+BUNDLESIZE_GITHUB_TOKEN as the name. 
+
+The last thing needed to kick-off continuous integration is a `.travis.yml`
+file, which tells Travis CI what to do. To speed things up, it is already
+included in the project and it specifies that the app is using NodeJS. 
+
+With this step, you’re all set up and bundlesize will warn you if your
+JavaScript ever goes over the budget. Even when you start off great, over time,
+as you add new features, kilobytes can pile up. With automated performance
+budget monitoring, you can rest easy knowing that it won’t go unnoticed. 
+
+## Try it out
+
+### Trigger your first bundlesize test
+
+To see how the app stacks up against the performance budget, add the code to the
+GitHub repo that you created in step 3. On Glitch, click your project name to go
+to Project options > Advanced options.
+
+<img src="advanced-options-button.png" class="screenshot" alt="Glitch Advanced options button">
+
+In the Advanced options menu, select Export to GitHub.
+
+<img src="export-to-github-button.png" class="screenshot" alt="Glitch Export to GitHub button">
+
+In the pop-up, enter your GitHub username and the name of the repo as
+"username/repo". 
+
+Glitch will export your app to a new branch named glitch. Create a new pull
+request by clicking the New pull request button on the homepage of the
+repository.
+
+You’ll now see status checks in progress on the pull request page.
+
+<img src="github-checks-in-progress.png" class="screenshot" alt="Github checks in progress">
+
+It won’t take long until all checks are done. Unfortunately, the cat voting app
+is a bit bloated and does not pass the performance budget check. The main bundle
+is 266KB and the budget is 170KB. 
+
+<img src="failed-bundlesize-check.png" class="screenshot" alt="Failed bundlesize check">
+
+### Optimize
+
+Luckily, there are some easy performance wins you can make by [removing unused
+code](https://web.dev/fast/remove-unused-code). There are two main imports in
+`src/index.js`:
+
+<pre class="prettyprint">
+import firebase from "firebase";
+import * as moment from 'moment';
+</pre>
+
+The app is using [Firebase Realtime
+Database](https://firebase.google.com/products/realtime-database/) to store the
+data, but it’s importing the entire firebase package which consists of a lot
+more than just a database (auth, storage, messaging etc.).
+
+Fix this by importing only the package that the app needs in the `src/index.js`
+file:
+
+<pre class="prettyprint">
+import firebase from "firebase";
+import firebase from 'firebase/app';
+import 'firebase/database';
+</pre>
+
+<div class="aside note">
+The <code>firebase/app</code> import, which sets up the API surface for each of
+the different services, is always required.
+</div>
+
+### Re-run test
+
+Since the source file has been updated, you need to run webpack to build the new
+bundle file.
+
+* Click the **Logs** button.
+
+<img src="logs-button.png" class="screenshot" alt="Logs button">
```

Comment:
I think you can switch these over to `<web-screenshot type="logs">`. It will look like a 404 in the preview server because the patch is still waiting to go to production.

Reply:
Done 👍